### PR TITLE
Loading Page

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [ ] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

File: sites/example-project/src/pages/+layout.svelte [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <script>
+	import { browser } from '$app/environment';
+	if (browser) {
+		const splash = document.querySelector('#__evidence_project_splash');
```
Comment (csjh) [inline sites/example-project/src/pages/+layout.svelte:4]:
```suggestion
		const splash = document.getElementById('__evidence_project_splash');
```

File: sites/example-project/src/pages/+layout.server.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ import md5 from 'blueimp-md5';
 import { GET as getSettings } from './api/customFormattingSettings.json/+server.js';
 import { GET as getPagesManifest } from './api/pagesManifest.json/+server.js';
 import { getStatusAndExtractQueries } from './extractQueries.server.js';
-export const prerender = true;
+import { dev } from '$app/environment';
+export const prerender = !dev;
```
Comment (csjh) [inline sites/example-project/src/pages/+layout.server.js:7]:
```suggestion

```

File: sites/example-project/src/app.html
Comment (csjh) [inline sites/example-project/src/app.html:1]: If we want a guarantee that the `__evidence_project_splash` won't render after the SK app takes over, we could have something like
```js
// app.html script tag
globalThis.removeSplash = () => {
  const splash = document.getElementById('__evidence_project_splash');
  if (splash) {
    splash.style.visibility = 'hidden';
  }
}
```
then in `+layout.svelte` `context="module"` script tag:
```js
globalThis.removeSplash?.();
```

Comment (mcrascal) [reply sites/example-project/src/app.html:1]: I like it, but I think it'd be breaking for projects w/ custom layouts.

**Comment by mcrascal** [comment]
@ItsMeBrianD I've got a blocking issues in here. It looks like dev mode chart loading states flicker through an error state instead of rendering their skeleton.

File: packages/ui/core-components/src/lib/organisms/layout/EvidenceDefaultLayout.svelte [hunk 3]
```diff
@@ -8,6 +8,13 @@
 	import BreadCrumbs from './BreadCrumbs.svelte';
 	import TableOfContents from './tableofcontents/TableOfContents.svelte';
 	import ErrorOverlay from './ErrorOverlay.svelte';
+	import { browser } from '$app/environment';
+
+	// Remove splash screen from app.html
+	if (browser) {
+		const splash = document.querySelector('#__evidence_project_splash');
```
Comment (csjh) [inline packages/ui/core-components/src/lib/organisms/layout/EvidenceDefaultLayout.svelte:15]:
```suggestion
		const splash = document.getElementById('__evidence_project_splash');
```

File: sites/example-project/src/pages/+error.svelte [hunk 4]
```diff
@@ -2,6 +2,14 @@
 	import { page } from '$app/stores';
 	import { Accordion, AccordionItem, CopyButton } from '@evidence-dev/core-components';
 
+	import { browser } from '$app/environment';
+
+	// Remove splash screen from app.html
+	if (browser) {
+		const splash = document.querySelector('#__evidence_project_splash');
```
Comment (csjh) [inline sites/example-project/src/pages/+error.svelte:9]:
```suggestion
		const splash = document.getElementById('__evidence_project_splash');
```

**Review by csjh — APPROVED** [review]

**Pull request merged**